x-pack/plugins/observability/public/components/alert_search_bar/alert_search_bar.tsx

```diff
@@ -41,6 +43,7 @@ export function ObservabilityAlertSearchBar({
   status,
 }: ObservabilityAlertSearchBarProps) {
   const toasts = useToasts();
+  const { uiSettings } = useKibana().services;
```

Comment:
@fkanout We should not use the `useKibana` here directly but rather provide the related services via `services` prop to ensure this dependency is clear for the consumers. (That was the reason that this component didn't have `useKibana` so far)
For more information about why we need to pass dependency like this, you can check this [presentation](https://docs.google.com/presentation/d/1Za98KuAI0S7DQrVR-FGMj-H62er7j8YmNY2rCe7PYHI/edit#slide=id.g11f0f180654_1_0).

Would you please create a PR to fix this issue?

cc @mgiota

Reply:
@maryam-saeidi Very good point, you are right we have services passed, so we can get uiSettings from there. Nice presentation!